src/components/datagrid/body/data_grid_body.tsx

```diff
@@ -359,6 +360,19 @@ export const EuiDataGridBody: FunctionComponent<EuiDataGridBodyProps> = (
     visibleRowCount,
   ]);
 
+  /**
+   * Handles scroll cells fully into view
```

Comment:
Derp, I was apparently mathing so hard I forgot how to word
```suggestion
   * Handles scrolling cells fully into view
```